public/editor/FeatureGallery.tsx

```diff
@@ -243,6 +244,10 @@ const FeatureSelector = ({
       });
     };
 
+    let colorFormat = gallerySectionConfig.colorFormat
```

Comment:
Nitpick that ideally I should have a lint rule to enforce but I guess I don't... but I prefer `const` for variables that don't change. Same with `presetColors` below.